app/config/config.neon

```diff
@@ -89,43 +89,44 @@ parameters:
     siteName: "ReCodEx"  # name of frontend (ReCodEx, or KSP for unique instance for KSP course)
     githubUrl: https://github.com/ReCodEx  # URL to GitHub repository of this project
     from: "ReCodEx <noreply@recodex.mff.cuni.cz>"  # sending email address
+    subjectPrefix: "ReCodEx - "
 
   failures:  # admin messages on errors
     emails:  # additional info for sending mails
       to: "Administrator <admin@example.com>"  # admin mail address
       from: %emails.from%  # source address
-      subjectPrefix: "ReCodEx Failure Report - "  # prefix of mail subject
+      subjectPrefix: "Failure Report -"  # prefix of mail subject
 
   assignmentNotifications:
     emails:
       from: %emails.from%
-      newAssignmentPrefix: "ReCodEx New Assignment Notification - "
-      assignmentDeadlinePrefix: "ReCodEx Assignment Deadline Is Behind the Corner - "
+      newAssignmentPrefix: "New Assignment -"
+      assignmentDeadlinePrefix: "Assignment Deadline Is Behind the Corner -"
       assignmentDeadlineThreshold: 1 day
 
   submissionNotifications:
     emails:
       from: %emails.from%
-      submissionEvaluatedPrefix: "ReCodEx Submission Evaluated Notification - "
+      submissionEvaluatedPrefix: "Submission Evaluated - "
 
   failureResolutionNotifications:
     emails:
       from: %emails.from%
-      failureResolvedPrefix: "ReCodEx Submission Failure resolved - "
+      failureResolvedPrefix: "Submission Failure Resolved - "
```

Comment:
If I'm not mistaken, these strings are already hardcoded as defaults in the email helpers. Shouldn't we move this to the example config or remove it completely?

Reply:
Well, let me just say that I am not really fan of the `Arrays::get("key", "default")` in code.
Because of its duplication :-) ... In my opinion all configurable items should be defined in global configuration (`config.neon`) and the ones which really needs to change on every deployment in `config.local.neon`.
Why in global configuration rather then in code? Because its more readable to have all configuration items somewhere on the one place, so the administrator can decide what to override and what not. If these items would not be there as you suggest, nobody would know that it can be changed, because only few people will actually read the code.